lib/oid4vp.js

```diff
@@ -557,7 +557,7 @@ function _matchesInputDescriptor({
   return true;
 }
 
-function _fromQueryByExampleQuery(credentialQuery) {
+export function _fromQueryByExampleQuery(credentialQuery) {
```

Comment:
This is a private function, so it shouldn't be exported -- unless it's being exported for testing purposes only, which it should be marked as such with a comment if so:

```suggestion
// exported for testing purposes only
export function _fromQueryByExampleQuery(credentialQuery) {
```